content/miq_dialogs/cloud_volume_autosde_order.yaml

```diff
@@ -0,0 +1,247 @@
+---
+description: Order Storage Volume
+buttons: submit,cancel
```

Comment:
This is failing:
```
ActiveModel::UnknownAttributeError: unknown attribute 'buttons' for MiqDialog.
/home/grare/adam/src/manageiq/manageiq/app/models/miq_dialog/seeding.rb:49:in `seed_record'
/home/grare/adam/src/manageiq/manageiq/app/models/miq_dialog/seeding.rb:13:in `block (2 levels) in seed'
/home/grare/adam/src/manageiq/manageiq/app/models/miq_dialog/seeding.rb:12:in `each'
/home/grare/adam/src/manageiq/manageiq/app/models/miq_dialog/seeding.rb:12:in `block in seed'
/home/grare/adam/src/manageiq/manageiq/app/models/miq_dialog/seeding.rb:9:in `seed'
```

I think what you want is:
```
:content:
  :buttons:
  - :submit
  - :cancel
```

like https://github.com/ManageIQ/manageiq/blob/master/product/dialogs/miq_dialogs/miq_provision_dialogs.yaml#L5-L8

Reply:
If I apply this patch it seeds:
```
adam@desktop:~/src/manageiq/manageiq-providers-autosde$ git diff
diff --git a/content/miq_dialogs/cloud_volume_autosde_order.yaml b/content/miq_dialogs/cloud_volume_autosde_order.yaml
index 0c0bfae..21847cb 100644
--- a/content/miq_dialogs/cloud_volume_autosde_order.yaml
+++ b/content/miq_dialogs/cloud_volume_autosde_order.yaml
@@ -1,9 +1,11 @@
 ---
+name: cloud_volume_autosde_order
 description: Order Storage Volume
-buttons: submit,cancel
-label: AutoSDE Dialog
-system: false
-dialog_tabs:
+:content:
+  :buttons:
+  - :submit
+  - :cancel
+  :dialogs:
   - description:
     display:
     label: Volume
```

Reply:
![image](https://user-images.githubusercontent.com/6840118/185206001-43e854df-0358-4296-8a79-75696cf65467.png)

Hi @agrare this yaml originally came from the **export dialog tool**, There is a better way to export the dialog?